src/pages/developers/tutorials/swap-any.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: Swap To Any Token
+---
+
+## Overview
+
+In the previous tutorial you've learned how to create an omnichain swap
+contracts that allows users to swap between tokens between connected
+blockchains. The target token was always on a connected blockchain.
+
+In this tutorial you will modify the swap example to support swapping tokens to
```

Comment:
Maybe it's worth adding an explanation of why it would be useful to store your assets as ZRC20 rather than immediately withdrawing immediately. For example:

If you are processing several inbound transactions, you could batch your withdrawals to minimize your crosschain fees and gas fees.

Reply:
Good point!

Reply:
7dc96c3